compiler/next/include/chpl/uast/ASTNode.h

```diff
@@ -394,9 +395,12 @@ class ASTNode {
     }
   }
 };
-
-
 } // end namespace uast
+template<> struct stringify<chpl::uast::ASTNode> {
+  std::string operator()(StringifyKind stringKind, const chpl::uast::ASTNode& stringMe) const {
+    return defaultStringify(stringKind, stringMe);
```

Comment:
Might be nicer to call ASTNode::dump here for now

Reply:
hmmm....dump writes the output and returns void. Stringify is expecting to return a string, so we'd need to change dump or do something different

Reply:
This is the kind of thing that makes taking an ostream appealing, as you could pass an ostringstream to collect the output and stringify could return it. And if stringify also used an ostream, it would just pass it through to dump (which makes my first sentence moot but supports both outputting and getting a string)

Reply:
I updated the template to use a stream as suggested, let me know if it is not implemented as imagined.